proto/api/v1/api.proto

```diff
@@ -794,11 +802,25 @@ service ProcessorService {
 message ListProcessorsRequest {
   repeated string parent_ids = 1;
 }
-
 message ListProcessorsResponse {
   repeated Processor processors = 1;
 }
 
+message InspectProcessorRequest {
+  enum Direction {
```

Comment:
I wasn't sure what's better: direction or type, but now I'm leaning more towards "type". It doesn't feel right to say that an inspection has a direction, but it can be of different types.

Reply:
Looks simpler now without the direction enum.

Reply:
Yup!